src/store/apiSlice.ts

```diff
@@ -45,6 +45,7 @@ export type Relationships = {
 };
 
 export type StoreResource<AttributeType> = {
+  id?: string;
```

Comment:
Why was this added?

Reply:
this should have not reached git. It was from an understanding of the store. Removed